CHANGELOG.md

```diff
@@ -81,7 +81,7 @@ These should not affect the functionality of the module.
 - Include ruby- prefix in hiera example. [\#109](https://github.com/voxpupuli/puppet-rvm/pull/109) ([rubys](https://github.com/rubys))
 - System user and group [\#104](https://github.com/voxpupuli/puppet-rvm/pull/104) ([stintel](https://github.com/stintel))
 - -- Added a smartparam to toggle managing the rvmrc based on osfamily [\#103](https://github.com/voxpupuli/puppet-rvm/pull/103) ([madelaney](https://github.com/madelaney))
-- fix 'Undefined variable "::rvm\_version"' [\#99](https://github.com/voxpupuli/puppet-rvm/pull/99) ([jangrewe](https://github.com/jangrewe))
```

Comment:
I don't think we need to change the changelog

Reply:
Reverted